packit/api.py

```diff
@@ -374,8 +374,22 @@ def status(self):
             logger.info(tabulate(ds_prs, headers=["ID", "Title", "URL"]))
         else:
             logger.info("Downstream PRs: No open PRs.")
-        status.get_dg_versions()
-        status.get_up_releases()
+
+        dg_versions = status.get_dg_versions()
+        if dg_versions:
+            for branch, dg_version in dg_versions.items():
```

Comment:
Is there a title meant to be here?

Reply:
@rpitonak this one still needs your attention